SimeonInstaller.ps1

```diff
@@ -822,30 +827,31 @@ CRLFOption=CRLFAlways
             Write-Information "Repository already exists - will not create"
         }
 
-        if (!$repo.defaultBranch -and $GetImportUrl) {
-            $importUrl = . $GetImportUrl
+        if (!$repo.defaultBranch -and $GetSourceUrl) {
+            Push-Location (Get-GitRepository (. $GetSourceUrl))
 
-            if ($importUrl) {
-                Write-Information "Importing repository contents from '$importUrl'"
+            if ($ClearRepositoryContentsOnCreate) {
+                # delete Source/Resources/Content
+                Write-Information "Empty Baseline Selected. Removing template code."
```

Comment:
I would just put "Clearing repository contents" - might be nire confusing with current message.